actpool/actqueue.go

```diff
@@ -139,23 +137,16 @@ func (q *actQueue) Put(act *action.SealedEnvelope) error {
 			}
 		}
 		q.updateFromNonce(nonce)
+		q.ap.removeInvalidActs([]*action.SealedEnvelope{actInPool})
```

Comment:
what's the purpose?

Reply:
it's to remove replaced action by same nonce from actpool

Reply:
just curious, if this is a bug, (invalid act not removed now), would some test fail/pass due to this change?

Reply:
It break the test at actpool_test.go:379 after the introducing subscriber and blob validation, because the missing call of onRemoved, the blobTx was wrongly rejected

Reply:
so should fix the test by correctly adding `onRemoved()`, but not this change?